packages/tooling/fast-tooling/src/message-system/message-system.spec.md

```diff
@@ -0,0 +1,131 @@
+# Message System
+
+## Introduction
+
+The Message System controls the navigation, data, data schemas (JSON schemas) and history of messages it recieves. It can also be used to pass custom messages between services that are registered to it.
```

Comment:
```suggestion
The Message System controls the navigation, data, data schemas (JSON schemas), and history of messages it receives. It can also be used to pass custom messages between services that are registered to it.
```